### DISCUSSION TITLE
Ball Demo

### DISCUSSION THREAD

**Discussion by ktimam** [jrouwe/JoltPhysics#878] [Q&A]
Is there any demo for creating a ball (football, basketball, tennis ball, etc...)? Tried to play around with the sphere example settings, like friction, etc... but couldn't get the right parameters for a realistic simulation!

**Comment by jrouwe** [comment]
Sorry, can't really help you with that, I never made a ball game. I imagine that restitution and angular damping will be the most important parameters (friction doesn't do much for a ball, look at the FrictionTest). But a football is not really like a rigid body, so I'm not surprised that you can't get it to work. You could try a soft body with a very small time step.

**Comment by ktimam** [comment]
Used the following settings to create a ball:

RefConst<Shape> sphere_shape = new SphereShape(0.5f);
BodyCreationSettings bcs(sphere_shape, Vec3((double)m_cxClient / 2.0, 2.0_r, (double)m_cyClient / 2.0), Quat::sIdentity(), EMotionType::Dynamic, Layers::MOVING);
bcs.mOverrideMassProperties = EOverrideMassProperties::CalculateInertia;
bcs.mMassPropertiesOverride.mMass = Prm.BallMass;
//bcs.mFriction = 0.5;
bcs.mLinearDamping = 0.15;
bcs.mRestitution = 0.6f;
